modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -277,6 +277,16 @@
 		"issueKey": "LPS-196652",
 		"to": "addAccountGroupRel(param#2#, param#0#, param#1#)"
 	},
+	{
+		"from": "AccountPortletKeys.COMMERCE_ACCOUNT_ADMIN",
+		"issueKey": "LPS-197053",
+		"to": "AccountPortletKeys.ACCOUNT_USERS_ADMIN"
+	},
+	{
+		"from": "AccountPortletKeys.COMMERCE_ACCOUNT_GROUP_ADMIN",
+		"issueKey": "LPS-197053",
+		"to": "AccountPortletKeys.ACCOUNT_GROUPS_ADMIN"
```

Comment:
@TamyrisTorres, when we don't change the class we don't add in the `from` and `to` attributes, because adds unnecessary code repetition. Should be on the `classNames` attribute.

Reply:
resolved!